package-imports

```diff
@@ -6,79 +6,48 @@ apparmor
 apt-transport-https
 arptables
 auditd
-awscli
-azure-vm-utils
 binutils
-bird2
 bsdextrautils
 btrfs-progs
-bubblewrap
-build-essential
 ca-certificates
-ceph
 chrony
 cifs-utils
 clamav
-cloud-guest-utils
 cloud-init
 conntrack
 containerd
-containernetworking-plugins
 cryptsetup
 curl
-devscripts
-dkms
-dnsmasq
 dnsutils
-docker.io
 dosfstools
 dracut
 dracut-live
 dracut-network
-dwarves
```

Comment:
I would like to have a driver build container with the typical build dependencies installed. 

Why not some debian container? I would like to install the correct kernel headers for my target Garden Linux (kernel) version.